apps/ledger-live-mobile/src/components/SelectDevice2/index.tsx

```diff
@@ -152,28 +163,14 @@ export default function SelectDevice({ onSelect, stopBleScanning }: Props) {
 
   const onAddNewPress = useCallback(() => setIsAddNewDrawerOpen(true), []);
 
-  const onPairDevices = useCallback(() => {
-    const navigateInput: NavigateInput<
-      MainNavigatorParamList,
-      NavigatorName.Manager
-    > = {
-      name: NavigatorName.Manager,
-      params: {
-        screen: ScreenName.Manager,
-        params: {
-          device: null,
-        },
-      },
-    };
-    navigation.navigate(ScreenName.BleDevicePairingFlow, {
-      areKnownDevicesDisplayed: true,
-      onSuccessAddToKnownDevices: true,
-      onSuccessNavigateToConfig: {
-        navigateInput,
-        pathToDeviceParam: "params.params.device",
-      },
-    });
-  }, [navigation]);
+  const openBlePairingFlow = useCallback(() => {
+    setIsPairingDevices(true);
+  }, []);
+
+  const closeBlePairingFlow = useCallback(() => {
+    setIsPairingDevices(false);
+    setIsAddNewDrawerOpen(false);
+  }, []);
```

Comment:
You **[don't need](https://reactjs.org/docs/hooks-reference.html#usestate)** `useCallback` since you're not changing the behavior based on any internal variable
>React guarantees that setState function identity is stable and won’t change on re-renders. This is why it’s safe to omit from the useEffect or useCallback dependency list.

Reply:
this is more to avoid rerenders in case we memo the underlying components (if we pass () => setIsPairingDevices(true) directly in the prop, it will always rerender when the parent rerenders, even if the component is memoed)